edgexfoundry/app-service-configurable
# refactor!: Replace internal topics from config with new constants

BREAKING CHANGE: Internal topics no longer configurable, except the base topic. Trigger topics for edgex-messagebus and external-mqtt now directly under Trigger section.

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

 TBD

## Testing Instructions
Run non-secure EdgeX stack with mqtt-broker
Build ASC from this branch
Run ASC app locally with each profile
Verify it receives event data as expected. Some error may occur after events received, but that is ok at this point
For External MQTT profile, verify subscribed to MQTT Broker successfully. 

## New Dependency Instructions (If applicable)
